BOILERPLATE_README.md

```diff
@@ -40,9 +40,10 @@ When running `mix` or `make` commands, it is important that these variables are
 
 1. Create both `.env.dev.local` and `.env.test.local` from empty values in [`.env.dev`](./.env.dev) and [`.env.test`](./.env.test)
 2. Install Mix and NPM dependencies with `make dependencies`
-3. Create and migrate the database with `mix ecto.setup`
-4. Compile the application with `make compile`
-5. Start the Phoenix server with `iex -S mix phx.server` with environment variables from `.env.dev` and `.env.dev.local`
+3. Generate values for mandatory secrets in [`.env.dev`](./.env.dev) with `mix phx.gen.secret`
+4. Create and migrate the database with `mix ecto.setup`
+5. Compile the application with `make compile`
```

Comment:
This should be `mix compile`. Should we include this at all? Running `iex -S mix phx.server` will compile it anyway.